ui/apps/platform/src/Containers/Clusters/ClusterRegistrationSecrets/ClusterRegistrationSecretTechPreviewAlert.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Alert } from '@patternfly/react-core';
+
+import ExternalLink from 'Components/PatternFly/IconText/ExternalLink';
+import { getVersionedDocs } from 'utils/versioning';
+import useMetadata from 'hooks/useMetadata';
+
+export default function ClusterRegistrationSecretTechPreviewAlert() {
+    const { version } = useMetadata();
+    return (
+        <Alert
+            isInline
+            component="p"
+            variant="info"
+            title="Cluster registration secrets are a Technology Preview feature"
+        >
+            Cluster registration secrets (or &ldquo;CRS&rdquo; for short) are a modern alternative
+            to init bundles and will at some point replace init bundles entirely. Cluster
+            registration secrets and init bundles differ in their specific usage semantics &mdash;
+            please consult the{' '}
```

Comment:
Bravo for finding existing text.

For your information, when text does not already exist, docs team advised me about OpenShift pattern.

UI has 14 results in 10 files:

```tsx
For more information, see{' '}
<ExternalLink>…</ExternalLink>
```